herddb-core/src/main/java/herddb/cluster/BookkeeperCommitLogManager.java

```diff
@@ -62,10 +62,13 @@
     private long ledgersRetentionPeriod = 1000 * 60 * 60 * 24;
     private long maxLedgerSizeBytes = 100 * 1024 * 1024 * 1024;
     private long maxIdleTime = 0;
+    private long bookkeeperClusterReadyWaitTime = 60_000;
+
     private ConcurrentHashMap<String, BookkeeperCommitLog> activeLogs = new ConcurrentHashMap<>();
 
     public BookkeeperCommitLogManager(ZookeeperMetadataStorageManager metadataStorageManager, ServerConfiguration serverConfiguration, StatsLogger statsLogger) {
         this.statsLogger = statsLogger;
+        this.bookkeeperClusterReadyWaitTime = serverConfiguration.getInt(ServerConfiguration.PROPERTY_BOOKKEEPER_WAIT_CLUSTER_READY_TIMEOUT, ServerConfiguration.PROPERTY_BOOKKEEPER_WAIT_CLUSTER_READY_TIMEOUT_DEFAULT);
```

Comment:
getLong? or convert bookkeeperClusterReadyWaitTime to int

Reply:
good catch